MOBILE_API.xml

```diff
@@ -4316,7 +4316,9 @@
     </struct>
 
     <struct name="WeatherData" since="5.1">
-        <param name="currentTemperature" type="Temperature" mandatory="false"/>
+        <param name="currentTemperature" type="Temperature" mandatory="false">
+            <description>This parameter may be populated with a forecasted temperature even when the high and low parameters are not available.</description>
```

Comment:
I think that something like "The central temperature depending on the context of the weather data. It could be the present temperature, the temperature of a future minute, the temperature of a future hour, or an average temperature of a future day, for example." would be more descriptive.

Reply:
I think the fact this parameter depends on context needs to be inferred by a developer. I worry if we explicitly state this people will then question if they can infer that other parameters depend on context, when it should be obvious by how that struct is used within the spec. Does that make sense?

I do really like the examples you gave and changed the description in https://github.com/smartdevicelink/rpc_spec/pull/289/commits/c1568c9128dc38bb1d12040e2bf1a31eb8624fa2, please let me know what you think.

Reply:
The only difference between what we wrote is that mine says that it could be those different things based on the context. I don't see something wrong with stating why that parameter could contain those different pieces of data (the context that the struct is used in) because I also think we do need to define exactly how the parameter should be used. Developers should not be putting two different pieces of data into the same slot or else a service consumer displaying this data may label it incorrectly. 

Also should something similar be placed onto `apparentTemperature`?

Reply:
Updated in https://github.com/smartdevicelink/rpc_spec/pull/289/commits/e0c3727709f7df0bc85e517ec0eb0e7f0de917c0

Reply:
I am not sure about `apparentTemperature`, but we could add a similar note replacing the word temperature with 'apparent temperature' and it would have a similar effect. Let me know what you think.

Reply:
I think that's fine